### DISCUSSION TITLE
为什么 mac 开始后没有反应啊

### DISCUSSION THREAD

**Discussion by lupercaliaj** [lisai9093/YYS#7] [Q&A] [answered]
我用的app store下的版本，窗口也在左上角 但是各个模式开始游戏都没有反应， 只有突破游戏才开始点 但也是一直不开始挑战。跟我游戏分辨率有关系吗

**Comment by lisai9093** [comment, marked as answer]
M1的mac我没有测试过所以大概率不能正常工作，因为分辨率和桌面版应该是不一样的。mac的测试只针对于利用crossover运行的桌面版。

**Reply by zhang-wangz** [reply]
你好，大佬你用的crossover是20嘛

**Reply by lisai9093** [reply]
我用的是22.1版本

**Reply by zhang-wangz** [reply]
好滴，我去更新一下试试

**Reply by zhang-wangz** [reply]
> 我用的是22.1版本

因为看大佬没用窗口句柄的方式，固定了位置是

**Reply by lisai9093** [reply]
对 固定位置截图的

**Reply by zhang-wangz** [reply]
> 对 固定位置截图的

你好，大佬，你是用win10容器安装的吗

**Reply by lisai9093** [reply]
win10/11都可以，一定要是64位

**Reply by zhang-wangz** [reply]
> win10/11都可以，一定要是64位

好的

**Reply by zhang-wangz** [reply]
<img width="1677" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/d03e622d-bbf4-4724-ac3a-98d6277ad912">

**Reply by zhang-wangz** [reply]
没反应诶

**Comment by zhang-wangz** [comment]
<img width="400" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/417c8ec1-76f5-4baa-8535-fe69570a4410">
你好，可以问一下这些设置嘛，我好像没反应启动起来

**Reply by lisai9093** [reply]
打开高分辨率模式试试，macos一般自带2倍DPI，我之前可能把自动缩放模式关闭了。。。

**Reply by zhang-wangz** [reply]
嘶，开启了高分辨率模式，好像还是没反应
<img width="1579" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/3f1567bc-4f90-4f51-959d-eeda3462a40d">
<img width="905" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/e7d4bf0a-b13d-4bc2-b53a-d79e7aeb8e15">

**Reply by lisai9093** [reply]
没有反应可能是要进到百鬼界面才可以。另一个可能的原因是脚本基于Windows的桌面版截图，所以有可能会和crossover的截图不一致导致无法识别。

**Reply by zhang-wangz** [reply]
都试过了，进去和没进去，其他的模式也没反应。。。，看大佬的简介还以为是兼容的

**Reply by lisai9093** [reply]
刚试了下百鬼没问题：
<img width="633" alt="image" src="https://github.com/lisai9093/YYS/assets/3047171/0bee975c-ad9d-4cb2-afd8-895e40bd70df">

**Reply by zhang-wangz** [reply]
能看看大佬你那个crossover的设置嘛

**Reply by lisai9093** [reply]
<img width="1253" alt="image" src="https://github.com/lisai9093/YYS/assets/3047171/5b7e69f6-cf34-40b9-bf2c-0a47cfa4d3c4">
如果还是不行可能是分辨率不对，或者游戏的画质设置也有可能

**Reply by zhang-wangz** [reply]
<img width="260" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/b910fb6b-8c9a-474d-b8eb-40970edf09d6">

**Reply by lisai9093** [reply]
画质感觉没问题，你把游戏截图试试，我看看分辨率对不对

**Reply by zhang-wangz** [reply]
<img width="566" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/42086dbe-8d26-48e5-8fd9-af0d202739dc">

**Reply by lisai9093** [reply]
我的脚本可以识别这张图。你把脚本再更新下试试

**Reply by zhang-wangz** [reply]
好

**Reply by zhang-wangz** [reply]
更新完之后，
<img width="1323" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/9121b1f1-d6f1-4313-8ab3-f6a84837feb5">
就这样卡着不动诶。。。 下面那个火箭一直不停的跳，现在是这样

**Reply by lisai9093** [reply]
还是和之前一样没有识别，你是不是有多个屏幕？

**Reply by zhang-wangz** [reply]
是的，但是使用的时候固定在了这个屏

**Reply by lisai9093** [reply]
如果是多个屏幕游戏必须在最左侧屏幕的左上角才可以

**Reply by zhang-wangz** [reply]
..还是卡着
<img width="1095" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/20df7037-ffd7-45e7-a781-9b820f3dc860">

**Reply by lisai9093** [reply]
我的意思是你有多个外接显示器么（包括ipad的投屏），有的话必须放在最左侧屏幕：

**Reply by zhang-wangz** [reply]
嗷嗷，那个没有的

> 我的意思是你有多个外接显示器么（包括ipad的投屏），有的话必须放在最左侧屏幕：

**Reply by lisai9093** [reply]
那就奇怪了。我去做一个debug模式你再试试看

**Reply by zhang-wangz** [reply]
> 那就奇怪了。我去做一个debug模式你再试试看

好

**Reply by lisai9093** [reply]
已经加了，你更新下。Debug模式下会显示脚本的截图，你看看截图有没有问题

**Reply by lisai9093** [reply]
如果还是不行的话可以换成官方的IDLE试试

**Reply by zhang-wangz** [reply]
<img width="1142" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/eaba860e-1bf5-446c-bceb-fc2a2c3cab85">
截的图是这样的，确实截图了

**Reply by zhang-wangz** [reply]
是因为我的屏幕大小问题吗，😂

**Reply by lisai9093** [reply]
截图不对，正常的截图和游戏窗口差不多大的：
<img width="1204" alt="image" src="https://github.com/lisai9093/YYS/assets/3047171/640f11de-85be-401d-b781-fd5415b887a2">

**Reply by lisai9093** [reply]
你的屏幕分辨率多少，感觉DPI没有开启。我测试的环境是DPI为2倍分辨率

**Reply by zhang-wangz** [reply]
> 你的屏幕分辨率多少，感觉DPI没有开启。我测试的环境是DPI为2倍分辨率

<img width="515" alt="image" src="https://github.com/lisai9093/YYS/assets/49056179/c51e1ad4-d287-4c98-a2f8-a281cc5f79bc">
我是这样的

**Reply by zhang-wangz** [reply]
> 你的屏幕分辨率多少，感觉DPI没有开启。我测试的环境是DPI为2倍分辨率

怎么设置诶，我好像没有动过的

**Reply by lisai9093** [reply]
这个应该没问题，DPI已经开启了，好奇怪啊

**Reply by zhang-wangz** [reply]
啊这。。
